src/constructs/loadbalancing/alb.test.ts

```diff
@@ -229,13 +183,12 @@ describe("The GuApplicationListener class", () => {
 
     new GuApplicationListener(stack, "ApplicationListener", {
       loadBalancer,
-      overrideId: false,
       defaultAction: ListenerAction.forward([targetGroup]),
       certificates: [{ certificateArn: "" }],
     });
 
-    const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
-    expect(Object.keys(json.Resources)).not.toContain("ApplicationListener");
+    expect(stack).not.toHaveResourceOfTypeAndLogicalId("AWS::ElasticLoadBalancingV2::Listener", "AppListener");
```

Comment:
Should this be `ApplicationListener`?

Reply:
Ah sorry, I see now that we are passing `AppListener` in for other tests, but not here. I think we could remove this assertion and just have the regex check below?